{{cookiecutter.project_name}}/server/settings/environments/development.py

```diff
@@ -15,6 +15,11 @@
 
 DEBUG = True
 
+ALLOWED_HOSTS = [
+    config('DOMAIN_NAME'),
```

Comment:
Is there a way to preserve a `django`'s default? Something like:

```python
ALLOWED_HOSTS = [
    *default_settings.ALLOWED_HOSTS,
    config('DOMAIN_NAME'),
    '0.0.0.0',
]
```

Reply:
I can just add `['localhost', '127.0.0.1', '[::1]']` from [this](https://docs.djangoproject.com/en/2.1/ref/settings/) documentation page.

Reply:
@sobolevn any news?

Reply:
Agree